classes/class-pmprodev-migration-assistant.php

```diff
@@ -340,7 +364,13 @@ private static function helper_import_to_options( $import_data ) {
 				continue;
 			}
 
-			update_option( $option_name, wp_kses_post( $option_value ) );
+			$option_value = maybe_unserialize( $option_value );
+
+			if ( ! is_object( $option_value ) ) {
```

Comment:
```suggestion
			if ( ! is_object( $option_value ) && ! is_array( $option_value ) ) {
```
User fields will be an array of objects when imported. We do not want to pass an array into wp_kses_post().